packages/web-components/fast-components/docs/color.md

```diff
@@ -0,0 +1,143 @@
+# Color
+FAST utilizes an adaptive color system that provides some unique advantages:
+1. Ensure text meets [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/) contrast requirements.
+2. Easily swap from light mode to dark, or anywhere in-between.
+3. Color themeing through palette tinting.
+4. Perceptually uniform UI across different surface colors.
+
+## Algorithmic colors
+To understand the FAST color system, the first thing to understand is that all of the named colors enumerated below are algorithmic; they are a product of the *designSystem* object in which they are calculated. In the documentation below, these algorithmic colors will be referred to as *recipes*.
```

Comment:
This makes it more concise on the start of the paragraph.  "The first thing to understand in the FAST color system is that all ..."

Reply:
Took most of the fluff away - try that.